pyproject.toml

```diff
@@ -22,6 +22,7 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = [
+  "Flask-WTF<2",
```

Comment:
given that only the auth app needs this, is there somewhere better to add this dependency?

Reply:
Can we create a new extra named `auth`?

Reply:
yeah this sounds good